app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,14 @@
+
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+div#intro {
+    margin-top: 100px;
+    width: 80%;
+    margin-left: 20%;
+}
+
+div#foot-contact {
+    position: fixed;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered bottom, position